### ISSUE TITLE
Global text changes in draft-ietf-ccamp-layer1-types-03 after group renaming

### ISSUE THREAD

**Issue by haomianzheng** [haomianzheng/IETF-ACTN-YANG-Model#44]
We have done the following group renaming: 
otn-label-restriction -> otn-label-range-info; 
otn-link-label -> otn-label-start-end; 
otn-path-label -> otn-label-hop

After the other issue is confirmed https://github.com/haomianzheng/IETF-ACTN-YANG-Model/issues/43, a global change would be needed for the whole draft, especially for the text in Section 4.3.

**Comment by sergiobelotti** [comment]
Reading version 4 of the draft, just uploaded last week, it seems you already provided alignment in the grouping names  in seciton 4.3 :
otn-label-restriction -> otn-label-range-info;
otn-link-label -> otn-label-start-end;
otn-path-label -> otn-label-hop
The only leaf that is not described is the usage of range-bitmap in the label-restriction list.

**Comment by italobusi** [comment]
@sergiobelotti : the range-bitmap in the label-restriction list is used as defined in te-types. Should we describe its usage in the layer1-types?

**Comment by sergiobelotti** [comment]
@italobusi : when I made the comment above I've already checked description of range-bitmap in te-types, for me absolutely insufficent to understand usage in the case of OTN. In the case there is flexible mapping of a LO-ODU into ODUk , the usage of bit-map for any entry is essential to understand which TSs are assigned to any TP realted to the same ODU type. So the answer is yes, we should exploit section 4.3 in the draft to describe range-bitmap usage in OTN.
